edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/SubscriptionProcessor.cs

```diff
@@ -113,21 +113,24 @@ async Task ProcessSubscription(string id, Option<ICloudProxy> cloudProxy, Device
         async void CloudConnectivityEstablished(object sender, EventArgs eventArgs)
         {
             Events.DeviceConnectedProcessingSubscriptions();
-            try
+            async Task ProcessIdentity(IIdentity identity)
             {
-                IEnumerable<IIdentity> connectedClients = this.ConnectionManager.GetConnectedClients().ToList();
-                foreach (IIdentity identity in connectedClients)
+                try
                 {
-                    try
-                    {
-                        Events.ProcessingSubscriptionsOnDeviceConnected(identity);
-                        await this.ProcessExistingSubscriptions(identity.Id);
-                    }
-                    catch (Exception e)
-                    {
-                        Events.ErrorProcessingSubscriptions(e, identity);
-                    }
+                    Events.ProcessingSubscriptionsOnDeviceConnected(identity);
```

Comment:
Can we add logs that indicate this processing is on the device connected event?

Reply:
I added "on device connected" to the end of this log to make it more clear. Is that what you meant?